packages/keyring-api/src/internal/types.ts

```diff
@@ -1,12 +1,21 @@
 import type { Infer, Struct } from '@metamask/superstruct';
 import { boolean, string, number } from '@metamask/superstruct';
 
-import { BtcAccountType, EthAccountType, KeyringAccountStruct } from '../api';
+import {
+  BtcAccountType,
+  EthAccountType,
+  KeyringAccountStruct,
+  SolAccountType,
+} from '../api';
 import { BtcP2wpkhAccountStruct } from '../btc/types';
 import { EthEoaAccountStruct, EthErc4337AccountStruct } from '../eth/types';
+import { SolEoaAccountStruct } from '../sol/types';
```

Comment:
Should be renamed `SolAccountDataAccountStruct`, but let's wait for @danroc's input on the name first!